can_tools/scrapers/official/federal/HHS/facility.py

```diff
@@ -72,40 +72,59 @@ def normalize(self, data: str) -> pd.DataFrame:
         df.loc[:, numeric_cols] = df.loc[:, numeric_cols].where(lambda x: x > 0, np.nan)
 
         # Create new columns that we need
-        df["inpatient_beds_used_covid_7_day_avg"] = df.eval(
-            "total_adult_patients_hospitalized_confirmed_covid_7_day_avg + "
-            "total_pediatric_patients_hospitalized_confirmed_covid_7_day_avg"
-        )
+        vars_to_compute_sum = {
+            "inpatient_beds_used_covid_7_day_sum": (
+                "total_adult_patients_hospitalized_confirmed_covid_7_day_sum",
+                "total_pediatric_patients_hospitalized_confirmed_covid_7_day_sum",
+            ),
+            "inpatient_beds_used_covid_7_day_coverage": (
+                "total_adult_patients_hospitalized_confirmed_covid_7_day_coverage",
+                "total_pediatric_patients_hospitalized_confirmed_covid_7_day_coverage",
+            ),
```

Comment:
Hrm...  I think summing the `sum`s and `coverage`s before computing the `sum / coverage` 7-day average may be problematic since `a/b + c/d` != `(a+c)/(b+d)`?

Not sure how big of an issue this in practice (if the `_coverage` numbers are identical which seems likely then it'll work out), but I think it'd be more correct to compute the `sum/coverage` average for adults and pediatrics separately and then sum them.

Reply:
Er, actually I think this will definitely be problematic.  Consider the common case where there are lots of adults (e.g. 100) and very few pediatrics (e.g. 0) and they both have 7 days of coverage, your `coverage` will end up being `14` instead of `7` and so you'll basically be dividing the adult 7-day average in half.

Reply:
:facepalm: :face_with_head_bandage: 

Thank you for catching this. Will update accordingly.

Reply:
@cc7768 once this change is ready I will review